feature-report/src/main/java/org/verapdf/features/pb/objects/PBImageXObjectFeaturesObject.java

```diff
@@ -133,14 +135,182 @@ public FeatureTreeNode reportFeatures(FeaturesCollection collection) throws Feat
 		return null;
 	}
 
+	private static byte[] inputStreamToByteArray(InputStream is) throws IOException {
+		ByteArrayOutputStream baos = new ByteArrayOutputStream();
+		int reads = is.read();
+		while (reads != -1) {
+			baos.write(reads);
+			reads = is.read();
+		}
+		return baos.toByteArray();
+	}
```

Comment:
private static byte[] inputStreamToByteArray(InputStream is) throws IOException {
		ByteArrayOutputStream baos = new ByteArrayOutputStream();
		byte[] bytes = new byte[1024];
		int length;
		while ((length = is.read(bytes)) != -1) {
			baos.write(bytes, 0, length);
		}
		return baos.toByteArray();
	}